.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateTimeAltExtractor.cs

```diff
@@ -335,7 +300,27 @@ private bool IsConnectorOrWhiteSpace(int start, int end, string text)
                 data = ExtractDate_DateRange(former, latter);
             }
 
-            return data;
+            if (data.Count == 0)
```

Comment:
Based on our offline chat, let's make sure this whole method follows the same principle. At the very lease, invert the check below so the method can be split in two separate behaviours.
Thanks.